src/lib/utils/series-utils.js

```diff
@@ -65,6 +65,42 @@ function collectSeriesTypesInfo(children) {
   return result;
 }
 
+export function getStackedData(children, attr) {
+  const childData = [];
+  const attr0 = `${attr}0`;
+  children.forEach((child, childIndex) => {
```

Comment:
Yes. Even it is empty the correct result is returned (please see `!attr` check on line 78). However, I moved it in the inner function to avoid misunderstanding.